### ISSUE TITLE
Tables not automatically created

### ISSUE THREAD

**Issue by MishiR** [MetricsGrimoire/MailingListStats#70]
I am running the command on MySQL
create database mlstats;

Tables are not automatically created

**Comment by gpoo** [comment]
The databases are not created by mlstats, they must be created manually.

**Comment by MishiR** [comment]
Yes I have come to know this, I installed MySql workbench when I could not retrieve the tables. Actually I read in documentation that tables are created when I execute the create mlstats statement.
“The MySQL backend requires the database already exists. Use the following command to create the database. The tables will be automatically created when mlstats runs”

**Comment by gpoo** [comment]
I don't follow.

The command:

`create database mlstats;`

is a MySQL one. Once you run `mlstats`, the tables should be created.
